spec/models/user_spec.rb

```diff
@@ -55,6 +55,21 @@
     expect(@user.contacts).to include(contact)
   end
 
+  it "should have no relationship to social accounts" do
+    expect(@user.social_accounts).to be_empty
+  end
+
+  it "can have many social accounts" do
+    user_with_social_accounts = FactoryBot.build(:user)
+    user_with_social_accounts.save
```

Comment:
shouldn't be necessary because you never actually read the database, do you?

Reply:
We need to save the parent before calling create. But I will change the call on the FactoryBot to create.

Reply:
That's a good idea!

Reply:
Now that you use `create` this line shouldn't be necessary anymore